### PR TITLE
Fixes for restoring and keeping views

### PR DESCRIPTION
**Pull request by nist-anka**

### CODE REVIEW

File: IbisLib/scenemanager.cpp [hunk 1]
```diff
@@ -623,6 +623,12 @@ void SceneManager::AddObjectUsingID( SceneObject * object, SceneObject * attachT
         this->SetupInAllViews( object );
         object->PreDisplaySetup();
```
Comment (drouin-simon) [inline IbisLib/scenemanager.cpp]: We should also take into account all non-system objects. For example, if the first object to be added is a PolyDataObject, this should also reset the view.

File: IbisLib/view.cpp [hunk 2]
```diff
@@ -89,22 +89,39 @@ void View::Serialize( Serializer * ser )
...
         camera->SetPosition( pos );
         camera->SetFocalPoint( fp );
-        camera->SetParallelScale(scale);
+        if( parallel )
+        {
+            camera->SetParallelScale(scale);
+        }
+        else
+        {
+            camera->SetViewUp( viewUp );
```
Comment (drouin-simon) [inline IbisLib/view.cpp]: ViewUp applies to 2d views too.

**Comment by nist-anka** [comment]
When I restore it in 2D, it does not work, so I have to work more on it.

**Comment by nist-anka** [comment]
Should be ok now

**Pull request merged**